keyboards/projectd/65/projectd_65_ansi/info.json

```diff
@@ -0,0 +1,236 @@
+            { "flags": 4, "matrix": [6, 1], "x": 125, "y": 10 },
+            { "flags": 4, "matrix": [7, 5], "x": 135, "y": 10 },
+            { "flags": 4, "matrix": [7, 3], "x": 150, "y": 10 },
+            { "flags": 1, "matrix": [2, 1], "x": 0, "y": 20 },
+            { "flags": 4, "matrix": [1, 2], "x": 17.5, "y": 20 },
+            { "flags": 4, "matrix": [2, 2], "x": 27.5, "y": 20 },
+            { "flags": 4, "matrix": [3, 2], "x": 37.5, "y": 20 },
+            { "flags": 4, "matrix": [4, 2], "x": 47.5, "y": 20 },
+            { "flags": 4, "matrix": [4, 3], "x": 57.5, "y": 20 },
+            { "flags": 4, "matrix": [5, 3], "x": 67.5, "y": 20 },
+            { "flags": 4, "matrix": [5, 2], "x": 77.5, "y": 20 },
+            { "flags": 4, "matrix": [6, 2], "x": 87.5, "y": 20 },
+            { "flags": 4, "matrix": [7, 2], "x": 97.5, "y": 20 },
+            { "flags": 4, "matrix": [8, 2], "x": 107.5, "y": 20 },
+            { "flags": 4, "matrix": [8, 3], "x": 117.5, "y": 20 },
+            { "flags": 4, "matrix": [8, 4], "x": 127.5, "y": 20 },
+            { "flags": 4, "matrix": [2, 6], "x": 150, "y": 20 },
+            { "flags": 4, "matrix": [0, 0], "x": 0, "y": 30 },
+            { "flags": 4, "matrix": [1, 4], "x": 22.5, "y": 30 },
+            { "flags": 4, "matrix": [2, 4], "x": 32.5, "y": 30 },
+            { "flags": 4, "matrix": [3, 4], "x": 42.5, "y": 30 },
+            { "flags": 4, "matrix": [4, 4], "x": 52.5, "y": 30 },
+            { "flags": 4, "matrix": [4, 5], "x": 62.5, "y": 30 },
+            { "flags": 4, "matrix": [5, 5], "x": 72.5, "y": 30 },
+            { "flags": 4, "matrix": [5, 4], "x": 82.5, "y": 30 },
+            { "flags": 4, "matrix": [6, 4], "x": 92.5, "y": 30 },
+            { "flags": 4, "matrix": [7, 4], "x": 102.5, "y": 30 },
+            { "flags": 4, "matrix": [8, 5], "x": 112.5, "y": 30 },
+            { "flags": 4, "matrix": [0, 7], "x": 122.5, "y": 30 },
+            { "flags": 4, "matrix": [1, 6], "x": 140, "y": 30 },
+            { "flags": 4, "matrix": [6, 3], "x": 150, "y": 30 },
+            { "flags": 4, "matrix": [0, 6], "x": 0, "y": 40 },
+            { "flags": 4, "matrix": [0, 5], "x": 12.5, "y": 40 },
+            { "flags": 4, "matrix": [0, 2], "x": 25, "y": 40 },
+            { "flags": 4, "x": 61.5, "y": 40 },
+            { "flags": 4, "x": 62.5, "y": 40 },
+            { "flags": 4, "matrix": [0, 1], "x": 65, "y": 40 },
+            { "flags": 4, "x": 67.5, "y": 40 },
+            { "flags": 4, "x": 68.5, "y": 40 },
+            { "flags": 4, "matrix": [3, 6], "x": 100, "y": 40 },
+            { "flags": 4, "matrix": [3, 3], "x": 112.5, "y": 40 },
+            { "flags": 4, "matrix": [0, 3], "x": 130, "y": 40 },
+            { "flags": 4, "matrix": [1, 5], "x": 140, "y": 40 },
+            { "flags": 4, "matrix": [2, 5], "x": 150, "y": 40 }
+        ]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.4",
+        "pid": "0x5319",
+        "vid": "0x3233"
```

Comment:
This vendor ID belongs to DuckyChannel, and I suspect that this board isn't theirs.
https://www.the-sz.com/products/usbid/index.php?v=0x3233&p=&n=

Please change the Vendor ID.  Generally, 0x4000 and higher isn't assigned, and the further past this, the better.

Reply:
yes, this keyboard is from ducky's production. So, we keep this VID.

Reply:
Well, there is no indication of that anywhere here, aside from the Vendor ID.